product-service/src/main/resources/batch-local.yml

```
@@ -0,0 +1,8 @@
+spring:
+  batch:
+    jdbc:
+      initialize-schema: always
+batch:
+  product:
+    cron: "*/30 * * * * *"
+    batchSize: 5
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)